extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -23,4 +23,12 @@
      */
     @ConfigItem(defaultValue = "60")
     public Optional<Duration> defaultTransactionTimeout;
+
+    /**
+     * The directory name of location of the transaction logs.
+     * If the value is not absolute then the directory is relative
+     * to the <em>user.dir</em> system property.
+     * */
+    @ConfigItem(name = "objectStoreDir", defaultValue = "ObjectStore")
```

Comment:
```suggestion
    @ConfigItem(defaultValue = "ObjectStore")
```
Don't set a name and use the conventions.

Reply:
I fixed that one. We use kebab case in Quarkus (so dashes). Best is to remove the name and let Quarkus handle it automatically. I also updated the test config file.